### DISCUSSION TITLE
width of document symbols window

### DISCUSSION THREAD

**Discussion by khjcph** [folke/trouble.nvim#506] [Q&A] [answered]
Hi :) Is there a way that I can set the width of the document symbols window when displaying it on the left? I use this for the key map:

```
{
  "<leader>cs",
  "<cmd>Trouble symbols toggle win.position=left<cr>",
  desc = "Symbols (Trouble)",
}
```

**Comment by ddanieltan** [comment]
I'm also looking for the same guidance to change the default width

**Comment by folke** [comment, marked as answer]
You can add `win.size=100`

**Reply by khjcph** [reply]
thanks!
